server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -77,6 +77,13 @@
             <distributed-cache name="default" />
 
             <replicated-cache name="repl" configuration="replicated"/>
+
+            <counters>
+                <strong-counter name="c1" initial-value="0" storage="VOLATILE">
+                    <lower-bound value="0" />
+                </strong-counter>
+                <weak-counter name="c2" initial-value="0" storage="VOLATILE" concurrency-level="32" />
+            </counters>
```

Comment:
I'm pretty certain that we don't want to have two counters on the server's default clustered configuration. @tristantarrant any thoughts?

Reply:
Why not @ryanemerson ? It is ok to have one strong and one weak counter IMHO.

Reply:
@vblagoje Because you are creating an additional distributed cache to store the counter values. This doesn't come for free in terms of resources locally or RPCs across the cluster.

Reply:
@ryanemerson got it, @tristantarrant wdys?

Reply:
actually, you are creating 2 additional caches - a replicated and a distributed - and they are created when the CacheManager is started.

Reply:
No predefined counters in the default configuration please. We should just have an empty counters container and that's it.
We could have some examples of counters in the example directory, but then we might also need a test to ensure that the example works as expected.

Reply:
Additionally, the three configurations (standalone.xml, clustered.xml and domain.xml) should be more or less "equivalent" (with the obvious limitation that standalone.xml cannot have clustered features).